pcsx2/GS/Renderers/HW/GSRendererNew.cpp

```diff
@@ -1440,7 +1439,12 @@ void GSRendererNew::DrawPrims(GSTexture* rt, GSTexture* ds, GSTextureCache::Sour
 	}
 	else
 	{
-		EmulateATST(m_conf.cb_ps, m_conf.ps, false);
+		float aref;
```

Comment:
Maybe initializing it to 0 to silence the warning?

Reply:
Better yet; initialize it with the old value, possible reducing a cbuffer update :)